Cartfile

```diff
@@ -1,4 +1,4 @@
-github "3sidedcube/ThunderTable" == 1.2.2
+github "3sidedcube/ThunderTable" "feature/screen_reader"
```

Comment:
@ryanbourneuk will change this to 1.2.3 when has been released!

Reply:
👌